### PR TITLE
Rename check & setupJdks to disambiguate task shortcuts 

### PR DESCRIPTION
**Pull request by crogoz**
## Before this PR

Context: https://pl.ntr/2pc
Users were using `gw cJ` to run the compileJava task. `checkJava` is now matching this shortcut as well.

## After this PR

Change task names to disambiguate used shortcuts 
==COMMIT_MSG==
Rename check & setupJdks to disambiguate task shortcuts 
==COMMIT_MSG==

### CODE REVIEW

File: gradle-jdks/src/main/java/com/palantir/gradle/jdks/ToolchainsPlugin.java [hunk 1]
```diff
@@ -127,7 +127,7 @@ public void apply(Project rootProject) {
                 .named(LifecycleBasePlugin.CHECK_TASK_NAME)
                 .configure(check -> check.dependsOn(checkJdksLifecycle));
 
-        rootProject.getTasks().register("setupJdks", SetupJdksTask.class, setupJdksTask -> {
+        rootProject.getTasks().register("setupGradleJdks", SetupJdksTask.class, setupJdksTask -> {
```
Comment (crogoz) [inline gradle-jdks/src/main/java/com/palantir/gradle/jdks/ToolchainsPlugin.java]: changed the name for consistency

Comment (felixdesouza) [reply gradle-jdks/src/main/java/com/palantir/gradle/jdks/ToolchainsPlugin.java]: do you have to change things in the intellij plugin?

Comment (CRogers) [reply gradle-jdks/src/main/java/com/palantir/gradle/jdks/ToolchainsPlugin.java]: Isn't this going to break excavators?

Comment (crogoz) [reply gradle-jdks/src/main/java/com/palantir/gradle/jdks/ToolchainsPlugin.java]: > Isn't this going to break excavators?

you are right, thanks for catching this! I reverted this rename. 

> do you have to change things in the intellij plugin?

the intellij plugin is calling directly the `gradle-jdks-setup.sh` script

**Review by CRogers — APPROVED** [review]

**Pull request merged**